src/pages/starter-kit/checkout/payment-usage.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Payment API usage
```

Comment:
Consider adding Javascript to the title to differentiate the content from the webAPI stuff